cache/cache_test.go

```diff
@@ -180,15 +180,15 @@ func hit(sample string) check {
 	return func(t *testing.T, c BlobCache) {
 		// test whole blob
 		key := digestFor(sample)
-		testChunk(t, c, key, 0, sample)
+		testSpan(t, c, key, 0, sample)
```

Comment:
While it is true that SOCI doesn't have a notion of "chunks", it is equally true that cache/* does not have a notion of "spans". These are unit tests of a blob cache. The things that a blob cache stores are "blobs". Or leave them as "chunks" if you prefer. But let's not confuse them with an unrelated concept from elsewhere in the code.

Reply:
I updated this to keeping "chunk".